pkg/materialize/source.go

```diff
@@ -23,6 +24,44 @@ func GetTableStruct(v []interface{}) []TableStruct {
 	return tables
 }
 
+func DiffTableStructs(arr1, arr2 []interface{}) []TableStruct {
+	var difference []TableStruct
+
+	for _, item1 := range arr1 {
+		found := false
+		for _, item2 := range arr2 {
+			if areEqual(item1, item2) {
+				found = true
+				break
+			}
+		}
+		if !found {
+			if diffItem, ok := item1.(map[string]interface{}); ok {
+				difference = append(difference, TableStruct{
+					Name:  diffItem["name"].(string),
+					Alias: diffItem["alias"].(string),
+				})
+			}
+		}
```

Comment:
To remove `found`, you need to break out of the outer loop, not just let the inner loop complete.
```suggestion
out:
	for _, item1 := range arr1 {
		for _, item2 := range arr2 {
			if areEqual(item1, item2) {
				break out
			}
		}
		if diffItem, ok := item1.(map[string]interface{}); ok {
			difference = append(difference, TableStruct{
				Name:  diffItem["name"].(string),
				Alias: diffItem["alias"].(string),
			})
		}
	}
```